src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,50 @@
+.stop-watch {
+  width: 80vmin;
+  height: 80vmin;
+  justify-content: center;
```

Comment:
Add 'display: flex;' to correctly use 'justify-content' and 'align-items' for centering.